README.md

```diff
@@ -38,7 +38,10 @@ This repo includes a fully functional example app featuring threads, reactions,
 git clone git@github.com:GetStream/stream-chat-android.git
 ```
 
-Next, download Android Studio and open up the `stream-chat-android` folder. You'll want to run the `stream-chat-android-sample` app.
+Next, download Android Studio and open up the `stream-chat-android-ui-components-sample` folder. You'll want to run the `stream-chat-android-ui-components-sample` app.
```

Comment:
I'm not sure if we should change to `stream-chat-android-ui-components-sample` at this point.

Reply:
I think this should wait until we officially publish the new UI library